wcivf/apps/elections/templates/elections/includes/_ld_election.html

```diff
@@ -1,33 +1,42 @@
 {% load static %}
 <script type="application/ld+json">
-{
-    "@context": "http://schema.org/",
-    "@type": "Event",
-    "url" : "{{ CANONICAL_URL }}{{ election.get_absolute_url }}",
-    "image": "{{ CANONICAL_URL }}{% static "dc_theme/images/logo.png" %}",
-    "startDate": "{{ election.election.start_time | date:'Y-m-d H:i' }}",
-    "endDate": "{{ election.election.end_time | date:'Y-m-d H:i' }}",
+    {
+        "@context": "http://schema.org/",
+        "@type": "Event",
+        "url": "{{ CANONICAL_URL }}{{ election.get_absolute_url }}",
+        "image": "{{ CANONICAL_URL }}{% static "
+        dc_theme / images / logo.png " %}",
+        "startDate": "{{ election.election.start_time | date:'Y-m-d H:i' }}",
+        "endDate": "{{ election.election.end_time | date:'Y-m-d H:i' }}",
 
-{% if election.election.election_type == 'parl' %}
+        {
+            %
```

Comment:
Something fruity has gone on here with the `{% if %}` statements, guessing it will have been an indenter. They will need to be on single lines though otherwise they will get rendered in the html